tests/Integration/Infrastructure/Controller/Regulation/Fragments/SaveRegulationGeneralInfoControllerTest.php

```diff
@@ -100,6 +106,27 @@ public function testFieldsTooLong(): void
         $this->assertSame('Cette chaîne est trop longue. Elle doit avoir au maximum 255 caractères.', $crawler->filter('#general_info_form_description_error')->text());
     }
 
+    public function testMissingReasonsAndVisas(): void
```

Comment:
(Détail) C'est plutôt "empty" (on a le droit de ne pas en mettre mais si on en fournit la valeur doit être non-vide)

```suggestion
    public function testEmptyReasonsAndVisas(): void
```